R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -308,6 +308,21 @@ numCyl <- summarize(groupBy(carsDF, carsDF$cyl), count = n(carsDF$cyl))
 head(numCyl)
 ```
 
+`groupBy` can be replaced with `cube` or `rollup` to compute subtotals across multiple dimensions.
```

Comment:
I keep forgetting there is one. I think we can add a few lines. This is actually a pretty neat feature.